src/lib/swapper/swappers/OsmosisSwapper/OsmosisSwapper.ts

```diff
@@ -284,8 +306,6 @@ export class OsmosisSwapper implements Swapper<ChainId> {
           }),
         )
 
-      ibcSellAmount = await pollForAtomChannelBalance(receiveAddress, osmoUrl)
```

Comment:
Discussed that one with @kaladinlight - we may want to bring the *util* as a follow-up for paranoia safety, *not* this assignment however, we never want to send the full balance.
Rather, we should poll until there's enough balance (or potentially poll for Tx completion if possible), in case the 5000ms timeout isn't enough for the first Tx to complete, which it should under normal conditions.

Note, multi-hop is going to wait for the first Tx completion before initiating the second, at hop-level, so this is a non-problem there